### PR TITLE
Polyglot po loader

### PR DESCRIPTION
**Pull request by ThieryMichel**
Add polyglot-po-loader package for webbpack, to load po and convert them to json
- [x] convert to mono repo
- [x] add loader
- [x] share common logic in common package
- [x] update README

### CODE REVIEW

**Comment by djhi** [comment]
See #9. Not sure it should be done in this PR though

**Comment by djhi** [comment]
Needs rebase btw

**Comment by ThieryMichel** [comment]
> See #9. Not sure it should be done in this PR though

I'd rather do this in a separate PR

File: package.json [hunk 1]
```diff
@@ -1,62 +1,20 @@
...
-    "react-scripts": "3.0.1"
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/marmelab/polyglot-po.git"
   },
-  "jest": {
-    "watchPathIgnorePatterns": ["fixtures"]
-  }
+  "author": "",
+  "license": "ISC",
```
Comment (Kmaschta) [inline package.json]: Wrong license IMO, it should be Apache 2

File: packages/common/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "dependencies": {
+    "lodash": "4.17.15"
+  },
+  "devDependencies": {
+    "@babel/core": "7.4.5",
+    "@babel/preset-env": "7.4.5",
+    "jest": "24.7.1"
+  },
+  "author": "",
+  "license": "ISC"
```
Comment (Kmaschta) [inline packages/common/package.json]: Wrong license again

File: packages/polyglot-po-loader/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "dependencies": {
+    "common": "^1.0.0",
+    "lodash": "4.17.11",
+    "pofile": "1.1.0",
+    "webpack": "4.35.2",
+    "webpack-cli": "3.3.6"
+  },
+  "author": "",
+  "license": "ISC"
```
Comment (Kmaschta) [inline packages/polyglot-po-loader/package.json]: bis license

File: packages/polyglot-po/package.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
+{
+  "name": "polyglot-po",
+  "version": "0.0.1",
+  "license": "MIT",
```
Comment (Kmaschta) [inline packages/polyglot-po/package.json]: Nope, wrong license again

**Pull request merged**